fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/Slot.java

```diff
@@ -30,4 +32,8 @@ public Slot toSlot() {
     public Slot withNullable(boolean newNullable) {
         throw new RuntimeException("Do not implement");
     }
+
+    public Slot withQualifier(List<String> qualifiers) {
+        throw new RuntimeException("Do not implement");
```

Comment:
"Not implemented" may be a better exception message here

Reply:
ok，I will change it.